tests/test_schema_generation.py

```diff
@@ -53,3 +54,14 @@ class TestStandardPythonSchemaGeneration(object):
     def test_standard_handling(self):
         assert get_input_schema(standard_py_func) == self.standard_sample_input_schema
         assert get_output_schema(standard_py_func) == self.standard_sample_output_schema
+
+
+class TestNestedSchemaGeneration(TestCase):
+    nested_sample_input_schema = json.loads(
+        resource_string(__name__, os.path.join('resources', 'sample_nested_input_schema.json')).decode('ascii'))
+    nested_sample_output_schema = json.loads(
+        resource_string(__name__, os.path.join('resources', 'sample_nested_output_schema.json')).decode('ascii'))
+
+    def test_nested_handling(self):
```

Comment:
I don't think this test should be passing, since the sample json files still don't have input0 and output0. Can you get it to fail before updating the json files, this could indicate an issue with our test infra

Reply:
Json file has been updated. Based on the discussion, now we take every key as `required`, keep 'unwrapped' items as `{"type": "object"}` and keep `additionalProperties` as before if every item in nested dict is 'unwrapped'

Reply:
Looks like the expected output file still needs to be updated, which again has me concerned that this test isn't actually validating what we're trying to check.

Reply:
It seems previous `assertCountEqual` didn't dig into nested items and compare them each by each. Now we sort nested json object and using ordinary `assert` to judge.